### PR TITLE
Adding timeout

### PR DESCRIPTION
**Pull request by axeldenker**
Adding optional connection time out to exp.runtime.start().
Defaults to 10 seconds, set in api_utils.py

### CODE REVIEW

File: exp/lib/api_utils.py [hunk 1]
```diff
@@ -4,6 +4,13 @@
 from . import config
 from . import credentials
 
+def timeout():
+  if config.get("timeout"):
+    timeout=config.get("timeout")
+  else:
+     timeout=10
+  return timeout
+
```
Comment (jrd261) [inline exp/lib/api_utils.py]: Can we rename this get_timeout?
Also, ```return config.get('timeout') or 10``` can shorten this to a single line

**Comment by CISC** [comment]
Maybe nitpicking here, but since you can't actually specify a None timeout it's not exactly optional is it? :)
